src/main/java/sirius/biz/codelists/jdbc/SQLCodeListExportJobFactory.java

```diff
@@ -32,7 +36,10 @@
 public class SQLCodeListExportJobFactory
         extends EntityExportJobFactory<SQLCodeListEntry, SmartQuery<SQLCodeListEntry>> {
 
-    private CodeListParameter codeListParameter = new CodeListParameter("codeList", "$CodeList").markRequired();
+    private static final CodeListParameter CODE_LIST_PARAMETER =
```

Comment:
wasn't there a reason to make parameters non static? internal translations or the like?

Reply:
there are also a lot of static parameters in other projects' import/export jobs, so I don't think that is an issue?

Reply:
Parameters on all jobs are now constants.